src/setup.py

```diff
@@ -48,6 +48,7 @@
     install_requires=[
         "enum34",
```

Comment:
If we're going to do that, it would probably make sense to give leapp-tool its own repo, distinct from the one for the integration tests and demo - `pipenv` is fairly opinionated about the one repo = one environment model, and at the moment we're not really abiding by that (hence the need for `pipsi` in the integration tests).